k8s/README.md

```diff
@@ -0,0 +1,43 @@
+# Kubernetes Dashboards Template
+
+Provided by: [bonitoo.io](.)
+
+**This template provides 2 basic Kubernetes dashboards:
+`Kubernetes Node Metrics` and `Kubernetes Inventory`. The K8S infrastrucure
+supports Google Cloud Platform, AWS as well as on-premise K8S environments.**
```

Comment:
supports Google Cloud Platform, AWS, ~as well as~ **and** on-premise K8S environments.